internal/peer/channel/create.go

```diff
@@ -38,11 +38,12 @@ func (e InvalidCreateTx) Error() string {
 	return fmt.Sprintf("Invalid channel create transaction : %s", string(e))
 }
 
+// deprecated
 func createCmd(cf *ChannelCmdFactory) *cobra.Command {
 	createCmd := &cobra.Command{
 		Use:   "create",
-		Short: "Create a channel",
-		Long:  "Create a channel and write the genesis block to a file.",
+		Short: "deprecated: Create a channel",
+		Long:  "deprecated: Use Orderer Service Node (OSN). Create a channel and write the genesis block to a file.",
```

Comment:
```suggestion
		Short: "[DEPRECATED] Create a channel",
		Long:  "[DEPRECATED] Use Orderer Service Node (OSN). Create a channel and write the genesis block to a file.",
```

I think it would be better to use the same expression for deprecated as for other commands.
https://hyperledger-fabric.readthedocs.io/en/latest/commands/configtxgen.html

Reply:
@satota2 Thank you for your comments. 
I fixed them.

If you need to make such changes somewhere else, please tell me where. I do not know all the places.